apps/projects/contracts/test/BountiesEvents.sol

```diff
@@ -3,32 +3,38 @@ pragma solidity ^0.4.24;
 
 interface BountiesEvents {
 
-  /*
-   * Functions
-   */
+    /*
+     * Functions
+     */
 
-  function fulfillBounty(
-    address _sender,
-    uint _bountyId,
-    address[]  _fulfillers,
-    string _data
-  ) external;
-  /*
-   * Events
-   */
+    function fulfillBounty(
+        address _sender,
+        uint _bountyId,
+        address[]  _fulfillers,
+        string _data
+    ) external;
 
-  event BountyIssued(uint _bountyId, address _creator, address [] _issuers, address [] _approvers, string _data, uint _deadline, address _token, uint _tokenVersion);
-  event ContributionAdded(uint _bountyId, uint _contributionId, address  _contributor, uint _amount);
-  event ContributionRefunded(uint _bountyId, uint _contributionId);
-  event ContributionsRefunded(uint _bountyId, address _issuer, uint [] _contributionIds);
-  event BountyDrained(uint _bountyId, address _issuer, uint [] _amounts);
-  event ActionPerformed(uint _bountyId, address _fulfiller, string _data);
-  event BountyFulfilled(uint _bountyId, uint _fulfillmentId, address  [] _fulfillers, string _data, address _submitter);
-  event FulfillmentUpdated(uint _bountyId, uint _fulfillmentId, address  [] _fulfillers, string _data);
-  event FulfillmentAccepted(uint _bountyId, uint  _fulfillmentId, address _approver, uint[] _tokenAmounts);
-  event BountyChanged(uint _bountyId, address _changer, address  [] _issuers, address  [] _approvers, string _data, uint _deadline);
-  event BountyIssuersUpdated(uint _bountyId, address _changer, address  [] _issuers);
-  event BountyApproversUpdated(uint _bountyId, address _changer, address [] _approvers);
-  event BountyDataChanged(uint _bountyId, address _changer, string _data);
-  event BountyDeadlineChanged(uint _bountyId, address _changer, uint _deadline);
+    function contribute(
+        address _sender,
+        uint _bountyId,
+        uint _amount
+    ) external payable;
+    /*
+     * Events
+     */
+
+    event BountyIssued(uint _bountyId, address _creator, address [] _issuers, address [] _approvers, string _data, uint _deadline, address _token, uint _tokenVersion);
+    event ContributionAdded(uint _bountyId, uint _contributionId, address  _contributor, uint _amount);
+    event ContributionRefunded(uint _bountyId, uint _contributionId);
+    event ContributionsRefunded(uint _bountyId, address _issuer, uint [] _contributionIds);
+    event BountyDrained(uint _bountyId, address _issuer, uint [] _amounts);
+    event ActionPerformed(uint _bountyId, address _fulfiller, string _data);
+    event BountyFulfilled(uint _bountyId, uint _fulfillmentId, address  [] _fulfillers, string _data, address _submitter);
```

Comment:
Some spacing between array brackets and types seem wrong with inconsistently 1 or 2 spaces, maybe we should try to be consistent about it, I vote for no spacing between type and brackets
```suggestion
    event BountyFulfilled(uint _bountyId, uint _fulfillmentId, address[] _fulfillers, string _data, address _submitter);
```

Some for the rest of lines on this block